src/main/java/Parser.java

```diff
@@ -0,0 +1,130 @@
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Parser {
+    protected Storage file;
+    protected TaskList tList;
```

Comment:
As this class isn't extended by any other class, would a private modifier be more suitable here?

Reply:
This is incorrect, the Parser class is used by the UI class.